### ISSUE TITLE
Question : how to configure IP allocation ?

### ISSUE THREAD

**Issue by carguel** [chef/kitchen-vcenter#24]
The knife vcenter driver supports following option in order to setup a DHCP address allocation:

```
--cips dhcp
```

Is it possible to manage same behavior for VMs created by kitchen-vcenter ?

**Comment by jjasghar** [comment]
The `knife-vsphere` gem supports that but not `knife-vcenter` unless i'm missing something.

The underlying gem for `knife-vsphere` that @swalberg has been doing amazing work with, is different then the REST calls that `knife-vcenter` use. It would require research and some development time.

**Comment by carguel** [comment]
@jjasghar You are right, I made an error in my first comment: cips option is supported in vsphere driver for knife, not vcenter driver. 

As kitchen-vcenter leverages https://github.com/vmware/vsphere-automation-sdk-ruby/, I wonder if it is possible to support same kind of allocation ? 

I quickly made a search on vmware/vsphere-automation-sdk-ruby code, and I found some references that seem linked with this allocation. 

You can have a look to class ```IpAllocationPolicy``` in [client/sdk/com/vmware/vcenter/ovf.rb](https://github.com/vmware/vsphere-automation-sdk-ruby/blob/e982da39e11c6b91b221eb6ec68bb2eda58cfa11/client/sdk/com/vmware/vcenter/ovf.rb).

I don't know if it is the proper option, nor how it could be integrated in kitchen-vcenter. Any idea ?
